### PR TITLE
Install tflite differently for ubuntu 20.04 FOGL-7550

### PR DESCRIPTION
**Pull request by deepanshu-yadav**

### CODE REVIEW

File: extras_install.sh [hunk 1]
```diff
@@ -43,11 +47,15 @@ if [ ${ID} = "mendel" ]; then
...
 
+if [ "${os_name}" = "Ubuntu" ] && [ "${os_version}" = "20.04" ]; then
+   python3 -m pip  install tflite_runtime==2.8.0
+else
 
-py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
-arch=$(uname -m)
-url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
-python3 -m pip  install $url
+   py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
```
Comment (ashish-jabble) [inline extras_install.sh:54]: `python3 -V` looks more useful or `python3 -V | awk '{print $2}'`

File: extras_install.sh [hunk 2]
```diff
@@ -43,11 +47,15 @@ if [ ${ID} = "mendel" ]; then
 
 fi
 
+if [ "${os_name}" = "Ubuntu" ] && [ "${os_version}" = "20.04" ]; then
+   python3 -m pip  install tflite_runtime==2.8.0
```
Comment (ashish-jabble) [inline extras_install.sh:51]: remove extra space b/w pip and install. Also add some comment about different version of tflite_runtime for different platforms.

File: extras_install.sh [hunk 3]
```diff
@@ -43,11 +47,15 @@ if [ ${ID} = "mendel" ]; then
...
+else
 
-py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
-arch=$(uname -m)
-url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
-python3 -m pip  install $url
+   py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
+   arch=$(uname -m)
+   url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
+   python3 -m pip  install $url
```
Comment (ashish-jabble) [inline extras_install.sh:57]: remove extra space b/w pip and install

File: extras_install.sh [hunk 4]
```diff
@@ -43,11 +47,15 @@ if [ ${ID} = "mendel" ]; then
...
+   python3 -m pip  install tflite_runtime==2.8.0
+else
 
-py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
-arch=$(uname -m)
-url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
-python3 -m pip  install $url
+   py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
+   arch=$(uname -m)
+   url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
```
Comment (ashish-jabble) [inline extras_install.sh:56]: Better to add more information in echo statements.

**Review by AmandeepArora — APPROVED** [review]

File: extras_install.sh [hunk 5]
```diff
@@ -48,13 +48,18 @@ if [ ${ID} = "mendel" ]; then
...
+   # For ubuntu 20.04 install tflite runtime == 2.8.0
+   python3 -m pip install tflite_runtime==2.8.0
 else
 
+   # For every other platform install tflite runtime by constraucting a url.
    py=$(python3 -V | awk '{print $2}' | awk -F. '{print $1 $2}')
+   # Get the python version for this platform
    arch=$(uname -m)
+   # Get the architecture for this platform.
    url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n $py; echo -n "-cp"; echo -n $py; echo -n "m-linux_"; echo -n ${arch}; echo -n ".whl")
```
Comment (AmandeepArora) [inline extras_install.sh:60]: You can probably combine multiple trailing echo statement with something like:

url=$(echo -n "https://github.com/google-coral/pycoral/releases/download/release-frogfish/tflite_runtime-2.5.0-cp"; echo -n "${py}-cp${py}m-linux_${arch}.whl")

**Pull request merged**